content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Replace the GIF_PATH line with</p><p> </p><pre class="ql-syntax" spellcheck="false">GIF_PATH = os.path.join(DIR, 'xray_image.gif')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Cool. Done